h/views/api/bulk/annotation_schema.json

```diff
@@ -33,8 +33,10 @@
             "description": "A list of fields to include in the output",
 
             "type": "array",
-            "items": {"type": "string"},
-            "enum": [["author.username", "group.authority_provided_id"]]
+            "items": {
+                "type": "string",
+                "enum": ["annotation.metadata", "author.username", "group.authority_provided_id"]
```

Comment:
Added the new value and also change the definition to be a enum of three values instead of just one (with two elements).

The code in the service seems to be written to take a any combination of these so not sure if the definition here was intentional.